cpu/esp32/include/sdk_conf.h

```diff
@@ -29,15 +29,34 @@ extern "C" {
 #endif
 
 /**
-* @name    Clock configuration
-* @{
-*/
+ * @name    Clock configuration
+ * @{
+ */
+
+#ifndef DOXYGEN
+/* Mapping of Kconfig defines to the respective enumeration values */
+#if CONFIG_ESP32_DEFAULT_CPU_FREQ_MHZ_2
+#define CONFIG_ESP32_DEFAULT_CPU_FREQ_MHZ   2
+#elif CONFIG_ESP32_DEFAULT_CPU_FREQ_MHZ_40
+#define CONFIG_ESP32_DEFAULT_CPU_FREQ_MHZ   40
+#elif CONFIG_ESP32_DEFAULT_CPU_FREQ_MHZ_80
+#define CONFIG_ESP32_DEFAULT_CPU_FREQ_MHZ   80
+#elif CONFIG_ESP32_DEFAULT_CPU_FREQ_MHZ_160
+#define CONFIG_ESP32_DEFAULT_CPU_FREQ_MHZ   160
+#elif CONFIG_ESP32_DEFAULT_CPU_FREQ_MHZ_240
+#define CONFIG_ESP32_DEFAULT_CPU_FREQ_MHZ   240
+#endif
+#endif
+
 /**
  * @brief   Defines the CPU frequency [values = 2, 40, 80, 160 and 240]
  */
 #ifndef CONFIG_ESP32_DEFAULT_CPU_FREQ_MHZ
 #define CONFIG_ESP32_DEFAULT_CPU_FREQ_MHZ   80
 #endif
+/**
+ * @brief   Mapping configured ESP32 default clock to CLOCK_CORECLOCK define
+ */
```

Comment:
In the `CLOCK_CORECLOCK`definition below, you could use the `MHZ` macro and avoid the use of the multiplication by 1000000. Same for esp8266.

Reply:
That's not my change :wink: but I'm not sure if this was not even intended, as this file is also included by SDK header files and could lead to name clashes with existing macro definitions in the SDK. I will have to check. To be honest, there should be as little RIOT specific as possible in this file in general.

Reply:
That's why we don't have any `include` in this file.

Reply:
That was just a suggestion, it can stay like that no problem